web/templates/guest_visible_users_message.hbs

```diff
@@ -0,0 +1,6 @@
+<p id="guest_visible_users_message">
+    {{t "Guests will be able to see"}} {{user_count}} {{t "users in their channels when they join."}}
```

Comment:
This is not the correct way to translate, we cannot concatenate translated strings. See https://zulip.readthedocs.io/en/latest/translating/internationalization.html.